src/openrct2/peep/GuestPathfinding.cpp

```diff
@@ -1338,7 +1328,7 @@ Direction OriginalPathfinding::ChooseDirection(const TileCoordsXYZ& loc, Peep& p
 
     permitted_edges &= 0xF;
     uint8_t edges = permitted_edges;
-    if (isThin && peep.PathfindGoal == goal)
+    if (isThin)
```

Comment:
Why has this changed

Reply:
The code became redundant with PathfindGoal no longer being updated through a global variable, however thinking about it, adding a getter method, and making PathfindGoal private within the struct (and adding a getter method) is probably a good idea to make sure only the setter is used to change the PathfindGoal and improving the overall codebase quality.